backend/app/services/auth.py

```
@@ -1,145 +1,43 @@
 from typing import Optional, Dict, Any
 from loguru import logger
 
 from app.core.database import get_db
-from app.core.config import settings
-from app.schemas import User, SubscriptionTier
+from app.models import User
 from app.services.user import UserService
 
 security = HTTPBearer()
 
 def verify_clerk_token(token: str) -> dict:
```

Ruff (0.14.11)

Unused function argument: `token`

(ARG001)